validated/dash_apps/finished_apps/daily_validation.py

```diff
@@ -239,6 +239,7 @@
 # Layout
 app.layout = html.Div(
     children=[
+        html.Div(id="stations_list", hidden=True),
```

Comment:
Why is this hidden.

Reply:
This is just the list of stations that comes through from the view (i.e. the stations the user has change permissions for). This doesn't need to be shown, but it's used in the callback function to populate the drop-down list, so it needs to be included in the layout